fastlane/Fastfile

```diff
@@ -36,7 +36,14 @@ ALPHA_VERSION_BUNDLE_IDENTIFIER = 'com.automattic.alpha.woocommerce'
 # Registered in our Enterprise account, for App Center / Installable Builds
 ALPHA_BUNDLE_IDENTIFIERS = [
   ALPHA_VERSION_BUNDLE_IDENTIFIER,
-  "#{ALPHA_VERSION_BUNDLE_IDENTIFIER}.storewidgets"
+  # This is part of the project but currently disabled because of issues
+  # accessing the Enterprise account and creating the app id.
+  #
+  # Such a setup is no good long term but we're obviously working to sort it
+  # out. In the short term, not including this bundle id might work, or it
+  # might break the build later on when code sign happens but can't find the
+  # provisioning profile. We'll soon find out...
+  # "#{ALPHA_VERSION_BUNDLE_IDENTIFIER}.storewidgets"
```

Comment:
As expected, the build failed during code signing:

<img width="1170" alt="image" src="https://user-images.githubusercontent.com/1218433/187376623-a622560e-0a4f-4f00-8a02-25c2ab8afcfd.png">

```
error: StoreWidgetsExtension has conflicting provisioning settings. StoreWidgetsExtension is automatically signed for development, 
but a conflicting code signing identity iPhone Distribution has been manually specified.
Set the code signing identity value to "Apple Development" in the build settings editor, or switch to manual signing in the Signing & Capabilities editor.
(in target 'StoreWidgetsExtension' from project 'WooCommerce')
```